app/assets/javascripts/reducers/article_finder.js

```diff
@@ -0,0 +1,69 @@
+import _ from 'lodash';
+import { RECEIVE_CATEGORY_RESULTS, CLEAR_FINDER_STATE, RECEIVE_ARTICLE_PAGEVIEWS, RECEIVE_ARTICLE_PAGEASSESSMENT, RECEIVE_ARTICLE_REVISION, RECEIVE_ARTICLE_REVISIONSCORE } from "../constants";
+
+const initialState = {
+  articles: [],
+  loading: false
+};
+
+export default function articleFinder(state = initialState, action) {
+  switch (action.type) {
+    case CLEAR_FINDER_STATE: {
+      return {
+        articles: [],
+        loading: false
+      };
+    }
+    case RECEIVE_CATEGORY_RESULTS: {
+      let newStateArticles = state.articles.map(article => ({ ...article }));
```

Comment:
You should be able to do `newStateArticles = { ...state.articles }`. With that pattern, unchanged `article` objects get reused, which should prevent unnecessary rerendering of ArticleFinderRow.